lib/gds_api/locations_api.rb

```diff
@@ -0,0 +1,53 @@
+require_relative "base"
+require_relative "exceptions"
+
+class GdsApi::LocationsApi < GdsApi::Base
+  def local_custodian_code_for_postcode(postcode)
+    response = get_json("#{base_url}/postcode/#{CGI.escape postcode}.json")
+    location = Location.new(response) unless response.nil?
+    location.local_custodian_code
+  end
+
+  def coordinates_for_postcode(postcode)
+    response = get_json("#{base_url}/postcode/#{CGI.escape postcode}.json")
+    location = Location.new(response) unless response.nil?
+    {
+      "latitude" => location.lat,
```

Comment:
I'm not sure if this is the format that we want to return.

Reply:
Putting a `puts response.body` after [this line](https://github.com/alphagov/locations-api/blob/ab071ebcd941089c8e7e63e89dc8f01e63144965/spec/requests/v1/locations_spec.rb#L33) in Locations API, I can retrieve some example JSON that Locations API actually returns:

```
[
  {"postcode":"E1 8QS","address":"1, WHITECHAPEL HIGH STREET, LONDON, E1 8QS","latitude":51.5144547,"longitude":-0.0729933,"local_custodian_code":5900},
  {"postcode":"E1 8QS","address":"2, WHITECHAPEL HIGH STREET, LONDON, E1 8QS","latitude":51.5144548,"longitude":-0.0729934,"local_custodian_code":5900}
]
```

So there are two things I think need changing here:

1. Locations API returns an array. A limitation of Mapit was that it only returned one result; we'll want to return all results, i.e. `[ { "latitude" => ... }, { "latitude" => ...} ]`.
2. Locations API already returns a very flat structure, so there's no need to define the `Location` helper class. We can just return what we need directly, i.e. `{ "latitude" => response[n]["latitude"], ... }` etc.

(It may seem overly complex but several buildings can share the same postcode, so they can and do have different lat/long coordinates and local custodian codes. It'll be up to the frontend applications that use GdsApiAdapters to decide what to do with the array of results).